docs/csharp/language-reference/keywords/interface.md

```diff
@@ -27,7 +26,9 @@ An interface can be a member of a namespace or a class. An interface declaration
 - [Indexers](../../programming-guide/indexers/using-indexers.md)
 - [Events](event.md)
 
-These preceding member declarations typically do not contain a body. Beginning with C# 8.0, an interface member may declare a body. This is called a *default implementation*. Members with bodies permit the interface to provide a "default" implementation for classes and structs that don't provide an overriding implementation. In addition, beginning with C# 8.0, an interface may include:
+## Default interface members
+
+These preceding member declarations typically don't contain a body. Beginning with C# 8.0, an interface member may declare a body. Member body=ies in an interface are the *default implementation*. Members with bodies permit the interface to provide a "default" implementation for classes and structs that don't provide an overriding implementation. In addition, beginning with C# 8.0, an interface may include:
```

Comment:
```suggestion
These preceding member declarations typically don't contain a body. Beginning with C# 8.0, an interface member may declare a body. Member bodies in an interface are the *default implementation*. Members with bodies permit the interface to provide a "default" implementation for classes and structs that don't provide an overriding implementation. In addition, beginning with C# 8.0, an interface may include:
```

Also, I'm not sure how I'd change the opening sentence, but it's a bit jarring as it's starting a section/paragraph. At first, I assumed it was talking about a previous bit of content, but after reading it several times I realized that "These preceding member declarations" was referring to the `static abstract` and `static virtual` declarations, right?